types/node/v0/index.d.ts

```diff
@@ -1867,32 +1867,27 @@ declare module "assert" {
             expected: any;
             operator: string;
             generatedMessage: boolean;
+            code: 'ERR_ASSERTION';
 
-            constructor(options?: {message?: string; actual?: any; expected?: any;
-                                  operator?: string; stackStartFunction?: Function});
+            constructor(options?: {
+                message?: string; actual?: any; expected?: any;
+                operator?: string; stackStartFn?: Function
+            });
         }
 
-        export function fail(actual?: any, expected?: any, message?: string, operator?: string): never;
+        export function fail(actual: any, expected: any, message: any, operator: string): never;
```

Comment:
I think this change is not needed or an overload `fail(message?: string)` should be added.